### DISCUSSION TITLE
Opening new tab(ctrl + left click) in q-expansion-item not working with router.push(), but working with to method

### DISCUSSION THREAD

**Discussion by vijayy13** [quasarframework/quasar#10054] [General - Components / Directives / etc]
On clicking ctrl + left click in any of my sidebar pages, I want my page to be opened in a new tab. This works very well with q-expansion-item (to method), but when I change it to @click func => (router.push(name: 'test')) it seems to be not working...I can bend it to work by using router.resolve() but not sure what happened here. Can you anyone please help me in this ?

Page opened in new tab when clicking ctrl + left click:
  <q-item
        clickable
        v-ripple
        :active="$route.name === 'foo'"
        active-class="my-menu-link"
        :to="{ name: 'foo' }"
        dense
      >
<q-item
        clickable
        v-ripple
        :active="$route.name === 'foo'"
        active-class="my-menu-link"
         @click="router.push({name: 'foo'})"
        dense
      >
but this seems to be not working

**Comment by pdanpdan** [comment]
You have to check click event's keyboard modifyers (shift, Ctrl...) and implement what you want to happen (router push or openUrl...)

**Comment by vijayy13** [comment]
Yes I did make it to work by using the event modifier. But not sure how it's working with q-expansion-item's to method?

**Comment by pdanpdan** [comment]
It renders a link (<a>) and the rest is the native browser implementation.

**Comment by vijayy13** [comment]
ok thank you. Made it to work by using @click.ctrl="window.open(this.$router.resolve({ name: pathName }).href, "_blank")".
